vpr/src/draw/search_bar.cpp

```diff
@@ -96,14 +98,33 @@ void search_and_highlight(GtkWidget* /*widget*/, ezgl::application* app) {
             return;
         }
 
-        highlight_blocks((ClusterBlockId)block_id);
+        highlight_cluster_block((ClusterBlockId)block_id);
     }
 
     else if (search_type == "Block Name") {
         std::string block_name = "";
         ss >> block_name;
 
```

Comment:
Good to comment the overall logic.  Check if it is an atom name; if not, check if it is a cluster name.